SConstruct

```diff
@@ -1168,6 +1168,7 @@ else:
 			"/FC", # display full paths in diagnostics
 			"/EHsc", # catch c++ exceptions only
 			"/MP",  # enable multiprocessing of builds
+			"/std:$CXXSTD",
```

Comment:
On line 96, we have this :

```"c++11" if Environment()["PLATFORM"] != "win32" else ""```

I guess now we should drop the conditional there and just use "c++11" for all platforms?

Reply:
Yes, fixed in https://github.com/ImageEngine/cortex/pull/1224/commits/755e9ef589ee7949dc69b6fad97fe7497513a02c